recon-enumeration/activities/not-so-random/README.md

```diff
@@ -1,14 +1,8 @@
 # Name: Not So Random
 
-## Description
-
-Get the flag from [url]http://ctf-06.security.cs.pub.ro:8000/[/url].
-
-Score: 150
-
 ## Hints
```

Comment:
For session 07 we used something like:

Name:
---------

Here we have:

# Name.

We should follow the same rules for all README files, I think this one with headers is better. What do you think?
If you agree, please make a PR with changes for all READMEs in session 07.